src/main/java/io/github/hielkemaps/racecommand/race/Race.java

```diff
@@ -46,18 +52,44 @@ public Race(UUID owner, String name) {
 
     public void start() {
         place = 1;
+        huntedPlace = 1;
         finishedPlayers = new ArrayList<>();
+        huntedPlayers = new ArrayList<>();
         results = new ArrayList<>();
         isStarting = true;
 
         AtomicInteger seconds = new AtomicInteger(countDown);
 
         sendMessage(Main.PREFIX + "Starting race...");
 
+        for(UUID uuid : players) {
+            Player player = Bukkit.getPlayer(uuid);
+            player.removeScoreboardTag("hunter");
+            player.removeScoreboardTag("runner");
+        }
+
+        //Set hunter for manhunt races
+        if(variantType() == "manhunt") {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [ES: Comparison of String objects using == or != (ES_COMPARING_STRINGS_WITH_EQ)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq).
This code compares java.lang.String objects for reference equality using the == or != operators. Unless both strings are either constants in a source file, or have been interned using the String.intern() method, the same string value may be represented by two different String objects. Consider using the equals(Object) method instead.